modules/jflip.cc

```diff
@@ -0,0 +1,291 @@
+/*
+ * A skeleton for jill modules (jack clients using the jill framework). Creates
+ * an input and output port, but doesn't do anything. To customize:
+ *
+ * 1. Replace "modname" with the name of your module.
+ * 2. Add variables for configurable options in modname_options class
+ * 3. Edit modname_options constructor to define commandline options
+ * 4. Edit process() for the realtime logic.
+ * 5. Edit main() for startup and shutdown.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Copyright (C) 2010-2013 C Daniel Meliza <dan || meliza.org>
+ */
+#include <iostream>
+#include <signal.h>
+#include <boost/shared_ptr.hpp>
+
+// #include <sys/time.h>
+#include <stdlib.h>     /* srand, rand */
+#include <time.h>       /* time */
+#include <stdexcept>
+
+#include "jill/logging.hh"
+#include "jill/jack_client.hh"
+#include "jill/program_options.hh"
+
+// https://github.com/lemire/Code-used-on-Daniel-Lemire-s-blog/tree/master/2013/08/14/cpp
```

Comment:
You aren't using the more complex randomization algorithm any more, so these includes should be removed, not commented out.